backend/utils.ts

```diff
@@ -344,10 +344,6 @@ export const stringWithAbbreviationsProcessor = (string: string): string => {
   return establishmentName
 }
 
-export const isRedirectCaseLoad = (activeCaseLoadId: string): boolean => {
```

Comment:
This was purely taking into account the exemptions list which we are getting rid of in this PR